stable/gocd/templates/gocd-agent-deployment.yaml

```diff
@@ -27,7 +27,7 @@ spec:
         component: agent
     spec:
       serviceAccountName: {{ template "gocd.agentServiceAccountName" . }}
-      {{- if or .Values.agent.persistence.enabled .Values.agent.security.ssh.enabled }}
+      {{- if or .Values.agent.persistence.enabled (or .Values.agent.security.ssh.enabled .Values.agent.persistence.extraVolumes) }}
```

Comment:
@sheroy - thanks, had missed this while checking #9873 . Please check gocd-server-deployment.yaml. You need to add this check for volumeMounts in the container spec as well, I believe.

Reply:
@sheroy - Nevermind. I thought that the extraVolumes change at https://github.com/helm/charts/blame/master/stable/gocd/templates/gocd-agent-deployment.yaml#L116 was missing. Ignore the comment. I'll test this a bit later and merge if that's alright.

Reply:
Sure, that's fine. Thanks @varshavaradarajan.